app/controllers/lead_providers/guidance_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
I think you mentioned that the controllers etc. were copied from a different project in this github org. It would be good to mention the source in the PR, and ideally also in the relevant commit(s).

Reply:
I'll add the links to the originals in when I squash the commit?

Reply:
I personally dislike the squash options on github because https://timwise.co.uk/2021/03/15/github-rebase-and-squash-considered-harmful/

Reply:
I know....

Reply:
added source comments where applicable